packages/python-packages/api-stub-generator/ci.yml

```diff
@@ -0,0 +1,45 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - packages/python-packages/api-stub-generator
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - packages/python-packages/api-stub-generator
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tool-python.yml
+  parameters:
+    PythonVersion: '3.10.2'
+    PackagePath: 'packages/python-packages/api-stub-generator'
+    FeedName: 'public/azure-sdk-api-review-tools'
+    ArtifactName: 'apiviewparserpython'
+    PackageName: 'Python API View Parser'
+    TestSteps:
+    - template: /eng/pipelines/templates/steps/use-python-version.yml
+      parameters:
+        versionSpec: '3.10.2'
+
+    - script: |
+        python $(Build.SourcesDirectory)/eng/scripts/python_version_check.py
+      displayName: 'Verify Python APIView version consistency'
+
+    - script: |
+        cd $(Build.SourcesDirectory)/packages/python-packages/api-stub-generator
```

Comment:
Is the package name api-stub-generator as well? I'm mostly asking because we generally want to use a correctly namespaced name as we start to publish packages. I don't know what the actual conventionis for python but for .NET we use Microsoft.* and JS we use @azure-tools etc but something that indicates it is from our team or MS.

Reply:
It is api-stub-generator. We can make a separate change to update this name to azure-apiview-stub-generator

Reply:
Yes we should come up with some naming that is less generic.